espanol/canudos.md

```diff
@@ -57,3 +57,5 @@ el Consejero:
 [Un ejercito de angeles para ayudar a los guerreros.](angeles/angeles.md)
 
 [En el rincón donde dormía, el beatito...](beatito/beatito.md)
+
+[Se disculpó con Beatito y abandonó la sala para ocupar su lugar junto al emperador en el recibimiento](recibimiento/recibimiento.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.